### ISSUE TITLE
Linux build gives many warnings about unused -isystem flag.

### ISSUE THREAD

**Issue by ry** [denoland/deno#438]
```
clang++: warning: argument unused during compilation: '-isystem ../../buildtools/third_party/libc++abi/trunk/include' [-Wunused-command-line-argument]
```

**Comment by Pato91** [comment]
Hi @ry, I'm gonna start by saying that I'm really not at the same level of skill as you guys but I have been trying to look through the issues to try and support deno any way I can in my own small way. I'm a huge fan of the whole project and I often take time to try and look through its source.

Jumping to the point about this issue... I have been doing some reading and among the error messages within the many warnings, I saw that a certain file '../../tools/cfi/blacklist.txt' was missing and I decided to try and look up the chromium source specifically the file at [the blacklist.txt of the chromium source](https://chromium.googlesource.com/chromium/src/+/lkgr/tools/cfi/blacklist.txt).

I found the file and tried to add it manually to the '../../tools/cfi/' in deno's source directory and ran the build command again on Linux. Though I have linting warnings in the console, the build seems to run successfully without terminating in errors.

I also took a number of screenshots to show the kind of results I'm getting on my system.

I'm running the build on linux mint with the os-release info below
`NAME="Linux Mint"
VERSION="18.3 (Sylvia)"
ID=linuxmint
ID_LIKE=ubuntu
PRETTY_NAME="Linux Mint 18.3"
VERSION_ID="18.3"
HOME_URL="http://www.linuxmint.com/"
SUPPORT_URL="http://forums.linuxmint.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/linuxmint/"
VERSION_CODENAME=sylvia
UBUNTU_CODENAME=xenial
`
Adding the ../../tools/cfi/blacklist.txt
![with blacklist file](https://user-images.githubusercontent.com/29727441/43746111-e1aab14e-99ea-11e8-9d25-1ea10e258ea3.png)
![snapshot of the added folder](https://user-images.githubusercontent.com/29727441/43746108-e120b3b8-99ea-11e8-9c7c-6668f22af910.png)

The build has a bunch of warnings but works
![build with a bunch of error](https://user-images.githubusercontent.com/29727441/43746110-e17c8260-99ea-11e8-9544-9270760b09dc.png)

Build in progress
![build in progress](https://user-images.githubusercontent.com/29727441/43746109-e14e7a0a-99ea-11e8-9def-5bf26ece1c85.png)

Build finished
![build finished](https://user-images.githubusercontent.com/29727441/43746107-e0f29b40-99ea-11e8-8584-7dfde7221fcd.png)

**Comment by ry** [comment]
@Pato91 Thanks for looking into this! 

It seems that `cfi/blacklist.txt` is used for a sanitization tool ( [added to the config here](https://cs.chromium.org/chromium/src/build/config/sanitizers/BUILD.gn?type=cs&q=tools/.*/blacklist.txt+file:build&g=0&l=327), [docs](https://clang.llvm.org/docs/ControlFlowIntegrity.html)  ). I don't think this is the right way to remove this warning.... 

> I saw that a certain file '../../tools/cfi/blacklist.txt' was missing

Where did you see that?

**Comment by Pato91** [comment]
I usually rebase my local deno repo and run the normal `./tools/build.py` script that runs usually without any error but just some few linting warnings provided you have the right environment setup(if everything needed is installed correctly)

For this missing file 

> I saw that a certain file '../../tools/cfi/blacklist.txt' was missing,

 I ran the release build command `DENO_BUILD_MODE=release ./tools/build.py :deno` and the build failed with a number of block texts referencing the '-isystem' flag each ending with the missing `../../tools/cfi/blacklist.txt` line.

**Comment by ry** [comment]
> I ran the release build command DENO_BUILD_MODE=release ./tools/build.py :deno and the build failed with a number of block texts referencing the '-isystem' flag each ending with the missing ../../tools/cfi/blacklist.txt line.

Interesting. Can you try that again with `-j1 -v` flags and copy the output here (or to a gist if it's super huge)

**Comment by Pato91** [comment]
Used the `-j1 -v` flags (also having removed the `../../tools/cfi/blacklist.txt` I had added and this is the output:

`DENO_BUILD_PATH: /media/meraki/RELATED/Work/OpenSource/deno/out/release
[1/871] /usr/bin/ccache ../../third_party/llvm-build/Release+Asserts/bin/clang++ -MMD -MF obj/build_extra/flatbuffers/compiler_files/go_generator.o.d -DV8_DEPRECATION_WARNINGS -DUSE_NSS_CERTS=1 -DUSE_X11=1 -DOFFICIAL_BUILD -DCHROMIUM_BUILD -DFIELDTRIAL_TESTING_ENABLED -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -DCR_CLANG_REVISION=\"335864-1\" -D_LIBCPP_DISABLE_VISIBILITY_ANNOTATIONS -D_LIBCXXABI_DISABLE_VISIBILITY_ANNOTATIONS -DCR_LIBCXX_REVISION=332543 -DCR_LIBCXXABI_REVISION=331450 -DCR_SYSROOT_HASH=4e7db513b0faeea8fb410f70c9909e8736f5c0ab -DNDEBUG -DNVALGRIND -DDYNAMIC_ANNOTATIONS_ENABLED=0 -I../../third_party/flatbuffers/grpc -I../.. -Igen -I../../third_party/flatbuffers/include -fno-strict-aliasing --param=ssp-buffer-size=4 -fstack-protector -funwind-tables -fPIC -B../../third_party/binutils/Linux_x64/Release/bin -pthread -fcolor-diagnostics -fmerge-all-constants -Xclang -mllvm -Xclang -instcombine-lower-dbg-declare=0 -no-canonical-prefixes -flto=thin -fwhole-program-vtables -fcomplete-member-pointers -m64 -march=x86-64 -O2 -fno-ident -fdata-sections -ffunction-sections -fno-omit-frame-pointer -g1 -fsanitize=cfi-vcall -fsanitize-blacklist=../../tools/cfi/blacklist.txt -fsanitize=cfi-icall -fvisibility=hidden -Wheader-hygiene -Wstring-conversion -Wtautological-overlap-compare -Wall -Wno-unused-variable -Wno-missing-field-initializers -Wno-unused-parameter -Wno-c++11-narrowing -Wno-covered-switch-default -Wno-unneeded-internal-declaration -Wno-undefined-var-template -Wno-address-of-packed-member -Wno-nonportable-include-path -Wno-user-defined-warnings -Wno-unused-lambda-capture -Wno-null-pointer-arithmetic -Wno-enum-compare-switch -Wno-ignored-pragma-optimize -Wno-exit-time-destructors -Wno-header-hygiene -Wno-return-type -std=gnu++14 -fno-exceptions -fno-rtti -nostdinc++ -isystem../../buildtools/third_party/libc++/trunk/include -isystem../../buildtools/third_party/libc++abi/trunk/include --sysroot=../../build/linux/debian_sid_amd64-sysroot -fvisibility-inlines-hidden -c ../../third_party/flatbuffers/grpc/src/compiler/go_generator.cc -o obj/build_extra/flatbuffers/compiler_files/go_generator.o
[2/871] /usr/bin/ccache ../../third_party/llvm-build/Release+Asserts/bin/clang++ -MMD -MF obj/build_extra/flatbuffers/compiler_files/cpp_generator.o.d -DV8_DEPRECATION_WARNINGS -DUSE_NSS_CERTS=1 -DUSE_X11=1 -DOFFICIAL_BUILD -DCHROMIUM_BUILD -DFIELDTRIAL_TESTING_ENABLED -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -DCR_CLANG_REVISION=\"335864-1\" -D_LIBCPP_DISABLE_VISIBILITY_ANNOTATIONS -D_LIBCXXABI_DISABLE_VISIBILITY_ANNOTATIONS -DCR_LIBCXX_REVISION=332543 -DCR_LIBCXXABI_REVISION=331450 -DCR_SYSROOT_HASH=4e7db513b0faeea8fb410f70c9909e8736f5c0ab -DNDEBUG -DNVALGRIND -DDYNAMIC_ANNOTATIONS_ENABLED=0 -I../../third_party/flatbuffers/grpc -I../.. -Igen -I../../third_party/flatbuffers/include -fno-strict-aliasing --param=ssp-buffer-size=4 -fstack-protector -funwind-tables -fPIC -B../../third_party/binutils/Linux_x64/Release/bin -pthread -fcolor-diagnostics -fmerge-all-constants -Xclang -mllvm -Xclang -instcombine-lower-dbg-declare=0 -no-canonical-prefixes -flto=thin -fwhole-program-vtables -fcomplete-member-pointers -m64 -march=x86-64 -O2-fno-ident -fdata-sections -ffunction-sections -fno-omit-frame-pointer -g1 -fsanitize=cfi-vcall -fsanitize-blacklist=../../tools/cfi/blacklist.txt -fsanitize=cfi-icall -fvisibility=hidden -Wheader-hygiene -Wstring-conversion -Wtautological-overlap-compare -Wall -Wno-unused-variable -Wno-missing-field-initializers -Wno-unused-parameter -Wno-c++11-narrowing -Wno-covered-switch-default -Wno-unneeded-internal-declaration -Wno-undefined-var-template -Wno-address-of-packed-member -Wno-nonportable-include-path -Wno-user-defined-warnings -Wno-unused-lambda-capture -Wno-null-pointer-arithmetic -Wno-enum-compare-switch -Wno-ignored-pragma-optimize -Wno-exit-time-destructors -Wno-header-hygiene -Wno-return-type -std=gnu++14 -fno-exceptions -fno-rtti -nostdinc++ -isystem../../buildtools/third_party/libc++/trunk/include -isystem../../buildtools/third_party/libc++abi/trunk/include --sysroot=../../build/linux/debian_sid_amd64-sysroot -fvisibility-inlines-hidden -c ../../third_party/flatbuffers/grpc/src/compiler/cpp_generator.cc -o obj/build_extra/flatbuffers/compiler_files/cpp_generator.o
[3/871] /usr/bin/ccache ../../third_party/llvm-build/Release+Asserts/bin/clang++ -MMD -MF obj/build_extra/flatbuffers/compiler_files/code_generators.o.d -DV8_DEPRECATION_WARNINGS -DUSE_NSS_CERTS=1 -DUSE_X11=1 -DOFFICIAL_BUILD -DCHROMIUM_BUILD -DFIELDTRIAL_TESTING_ENABLED -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -DCR_CLANG_REVISION=\"335864-1\" -D_LIBCPP_DISABLE_VISIBILITY_ANNOTATIONS -D_LIBCXXABI_DISABLE_VISIBILITY_ANNOTATIONS -DCR_LIBCXX_REVISION=332543 -DCR_LIBCXXABI_REVISION=331450 -DCR_SYSROOT_HASH=4e7db513b0faeea8fb410f70c9909e8736f5c0ab -DNDEBUG -DNVALGRIND -DDYNAMIC_ANNOTATIONS_ENABLED=0 -I../../third_party/flatbuffers/grpc -I../.. -Igen -I../../third_party/flatbuffers/include -fno-strict-aliasing --param=ssp-buffer-size=4 -fstack-protector -funwind-tables -fPIC -B../../third_party/binutils/Linux_x64/Release/bin -pthread -fcolor-diagnostics -fmerge-all-constants -Xclang -mllvm -Xclang -instcombine-lower-dbg-declare=0 -no-canonical-prefixes -flto=thin -fwhole-program-vtables -fcomplete-member-pointers -m64 -march=x86-64 -O2 -fno-ident -fdata-sections -ffunction-sections -fno-omit-frame-pointer -g1 -fsanitize=cfi-vcall -fsanitize-blacklist=../../tools/cfi/blacklist.txt -fsanitize=cfi-icall -fvisibility=hidden -Wheader-hygiene -Wstring-conversion -Wtautological-overlap-compare -Wall -Wno-unused-variable -Wno-missing-field-initializers -Wno-unused-parameter -Wno-c++11-narrowing -Wno-covered-switch-default -Wno-unneeded-internal-declaration -Wno-undefined-var-template -Wno-address-of-packed-member -Wno-nonportable-include-path -Wno-user-defined-warnings -Wno-unused-lambda-capture -Wno-null-pointer-arithmetic -Wno-enum-compare-switch -Wno-ignored-pragma-optimize -Wno-exit-time-destructors -Wno-header-hygiene -Wno-return-type-std=gnu++14 -fno-exceptions -fno-rtti -nostdinc++ -isystem../../buildtools/third_party/libc++/trunk/include -isystem../../buildtools/third_party/libc++abi/trunk/include --sysroot=../../build/linux/debian_sid_amd64-sysroot -fvisibility-inlines-hidden -c ../../third_party/flatbuffers/src/code_generators.cpp -o obj/build_extra/flatbuffers/compiler_files/code_generators.o
[4/871] /usr/bin/ccache ../../third_party/llvm-build/Release+Asserts/bin/clang++ -MMD -MF obj/build_extra/flatbuffers/compiler_files/java_generator.o.d -DV8_DEPRECATION_WARNINGS-DUSE_NSS_CERTS=1 -DUSE_X11=1 -DOFFICIAL_BUILD -DCHROMIUM_BUILD -DFIELDTRIAL_TESTING_ENABLED -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -DCR_CLANG_REVISION=\"335864-1\" -D_LIBCPP_DISABLE_VISIBILITY_ANNOTATIONS -D_LIBCXXABI_DISABLE_VISIBILITY_ANNOTATIONS -DCR_LIBCXX_REVISION=332543 -DCR_LIBCXXABI_REVISION=331450 -DCR_SYSROOT_HASH=4e7db513b0faeea8fb410f70c9909e8736f5c0ab -DNDEBUG -DNVALGRIND -DDYNAMIC_ANNOTATIONS_ENABLED=0 -I../../third_party/flatbuffers/grpc -I../.. -Igen -I../../third_party/flatbuffers/include -fno-strict-aliasing --param=ssp-buffer-size=4 -fstack-protector -funwind-tables -fPIC -B../../third_party/binutils/Linux_x64/Release/bin -pthread -fcolor-diagnostics -fmerge-all-constants -Xclang -mllvm -Xclang -instcombine-lower-dbg-declare=0 -no-canonical-prefixes -flto=thin -fwhole-program-vtables -fcomplete-member-pointers -m64 -march=x86-64 -O2 -fno-ident -fdata-sections -ffunction-sections -fno-omit-frame-pointer -g1 -fsanitize=cfi-vcall -fsanitize-blacklist=../../tools/cfi/blacklist.txt -fsanitize=cfi-icall -fvisibility=hidden -Wheader-hygiene -Wstring-conversion -Wtautological-overlap-compare -Wall -Wno-unused-variable -Wno-missing-field-initializers -Wno-unused-parameter -Wno-c++11-narrowing -Wno-covered-switch-default -Wno-unneeded-internal-declaration -Wno-undefined-var-template -Wno-address-of-packed-member -Wno-nonportable-include-path -Wno-user-defined-warnings -Wno-unused-lambda-capture -Wno-null-pointer-arithmetic -Wno-enum-compare-switch -Wno-ignored-pragma-optimize -Wno-exit-time-destructors -Wno-header-hygiene -Wno-return-type -std=gnu++14 -fno-exceptions -fno-rtti -nostdinc++ -isystem../../buildtools/third_party/libc++/trunk/include -isystem../../buildtools/third_party/libc++abi/trunk/include --sysroot=../../build/linux/debian_sid_amd64-sysroot -fvisibility-inlines-hidden -c ../../third_party/flatbuffers/grpc/src/compiler/java_generator.cc -o obj/build_extra/flatbuffers/compiler_files/java_generator.o
[5/871] /usr/bin/ccache ../../third_party/llvm-build/Release+Asserts/bin/clang++ -MMD -MF obj/build_extra/flatbuffers/compiler_files/flatc.o.d -DV8_DEPRECATION_WARNINGS -DUSE_NSS_CERTS=1 -DUSE_X11=1 -DOFFICIAL_BUILD -DCHROMIUM_BUILD -DFIELDTRIAL_TESTING_ENABLED -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -DCR_CLANG_REVISION=\"335864-1\" -D_LIBCPP_DISABLE_VISIBILITY_ANNOTATIONS -D_LIBCXXABI_DISABLE_VISIBILITY_ANNOTATIONS -DCR_LIBCXX_REVISION=332543 -DCR_LIBCXXABI_REVISION=331450 -DCR_SYSROOT_HASH=4e7db513b0faeea8fb410f70c9909e8736f5c0ab -DNDEBUG -DNVALGRIND -DDYNAMIC_ANNOTATIONS_ENABLED=0 -I../../third_party/flatbuffers/grpc -I../.. -Igen -I../../third_party/flatbuffers/include -fno-strict-aliasing --param=ssp-buffer-size=4 -fstack-protector -funwind-tables -fPIC -B../../third_party/binutils/Linux_x64/Release/bin -pthread -fcolor-diagnostics -fmerge-all-constants -Xclang -mllvm -Xclang -instcombine-lower-dbg-declare=0 -no-canonical-prefixes -flto=thin -fwhole-program-vtables -fcomplete-member-pointers -m64 -march=x86-64 -O2 -fno-ident -fdata-sections -ffunction-sections -fno-omit-frame-pointer -g1 -fsanitize=cfi-vcall -fsanitize-blacklist=../../tools/cfi/blacklist.txt -fsanitize=cfi-icall -fvisibility=hidden -Wheader-hygiene -Wstring-conversion -Wtautological-overlap-compare -Wall -Wno-unused-variable -Wno-missing-field-initializers -Wno-unused-parameter -Wno-c++11-narrowing -Wno-covered-switch-default -Wno-unneeded-internal-declaration -Wno-undefined-var-template -Wno-address-of-packed-member -Wno-nonportable-include-path -Wno-user-defined-warnings -Wno-unused-lambda-capture -Wno-null-pointer-arithmetic -Wno-enum-compare-switch -Wno-ignored-pragma-optimize -Wno-exit-time-destructors -Wno-header-hygiene -Wno-return-type -std=gnu++14 -fno-exceptions -fno-rtti -nostdinc++ -isystem../../buildtools/third_party/libc++/trunk/include -isystem../../buildtools/third_party/libc++abi/trunk/include --sysroot=../../build/linux/debian_sid_amd64-sysroot -fvisibility-inlines-hidden -c ../../third_party/flatbuffers/src/flatc.cpp -o obj/build_extra/flatbuffers/compiler_files/flatc.o
FAILED: obj/build_extra/flatbuffers/compiler_files/flatc.o
/usr/bin/ccache ../../third_party/llvm-build/Release+Asserts/bin/clang++ -MMD -MF obj/build_extra/flatbuffers/compiler_files/flatc.o.d -DV8_DEPRECATION_WARNINGS -DUSE_NSS_CERTS=1 -DUSE_X11=1 -DOFFICIAL_BUILD -DCHROMIUM_BUILD -DFIELDTRIAL_TESTING_ENABLED -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -DCR_CLANG_REVISION=\"335864-1\" -D_LIBCPP_DISABLE_VISIBILITY_ANNOTATIONS -D_LIBCXXABI_DISABLE_VISIBILITY_ANNOTATIONS -DCR_LIBCXX_REVISION=332543 -DCR_LIBCXXABI_REVISION=331450 -DCR_SYSROOT_HASH=4e7db513b0faeea8fb410f70c9909e8736f5c0ab -DNDEBUG -DNVALGRIND -DDYNAMIC_ANNOTATIONS_ENABLED=0 -I../../third_party/flatbuffers/grpc -I../.. -Igen -I../../third_party/flatbuffers/include -fno-strict-aliasing --param=ssp-buffer-size=4 -fstack-protector -funwind-tables -fPIC -B../../third_party/binutils/Linux_x64/Release/bin -pthread -fcolor-diagnostics -fmerge-all-constants -Xclang -mllvm -Xclang -instcombine-lower-dbg-declare=0 -no-canonical-prefixes -flto=thin -fwhole-program-vtables -fcomplete-member-pointers -m64 -march=x86-64 -O2 -fno-ident -fdata-sections -ffunction-sections -fno-omit-frame-pointer -g1 -fsanitize=cfi-vcall -fsanitize-blacklist=../../tools/cfi/blacklist.txt -fsanitize=cfi-icall -fvisibility=hidden -Wheader-hygiene -Wstring-conversion -Wtautological-overlap-compare -Wall -Wno-unused-variable -Wno-missing-field-initializers -Wno-unused-parameter -Wno-c++11-narrowing -Wno-covered-switch-default -Wno-unneeded-internal-declaration -Wno-undefined-var-template -Wno-address-of-packed-member -Wno-nonportable-include-path -Wno-user-defined-warnings -Wno-unused-lambda-capture -Wno-null-pointer-arithmetic -Wno-enum-compare-switch -Wno-ignored-pragma-optimize -Wno-exit-time-destructors -Wno-header-hygiene -Wno-return-type -std=gnu++14 -fno-exceptions -fno-rtti -nostdinc++ -isystem../../buildtools/third_party/libc++/trunk/include -isystem../../buildtools/third_party/libc++abi/trunk/include --sysroot=../../build/linux/debian_sid_amd64-sysroot -fvisibility-inlines-hidden -c ../../third_party/flatbuffers/src/flatc.cpp -o obj/build_extra/flatbuffers/compiler_files/flatc.o
clang++: error: no such file or directory: '../../tools/cfi/blacklist.txt'
ninja: build stopped: subcommand failed.`

**Comment by Pato91** [comment]
The fix works for the rest of the warnings but this one stays

`DENO_BUILD_PATH: /media/meraki/RELATED/Work/OpenSource/deno/out/release
[1/871] CXX obj/build_extra/flatbuffers/compiler_files/flatc.o
FAILED: obj/build_extra/flatbuffers/compiler_files/flatc.o
/usr/bin/ccache ../../third_party/llvm-build/Release+Asserts/bin/clang++ -MMD -MF obj/build_extra/flatbuffers/compiler_files/flatc.o.d -DV8_DEPRECATION_WARNINGS -DUSE_NSS_CERTS=1 -DUSE_X11=1 -DOFFICIAL_BUILD -DCHROMIUM_BUILD -DFIELDTRIAL_TESTING_ENABLED -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -DCR_CLANG_REVISION=\"335864-1\" -D_LIBCPP_DISABLE_VISIBILITY_ANNOTATIONS -D_LIBCXXABI_DISABLE_VISIBILITY_ANNOTATIONS -DCR_LIBCXX_REVISION=332543 -DCR_LIBCXXABI_REVISION=331450 -DCR_SYSROOT_HASH=4e7db513b0faeea8fb410f70c9909e8736f5c0ab -DNDEBUG -DNVALGRIND -DDYNAMIC_ANNOTATIONS_ENABLED=0 -I../../third_party/flatbuffers/grpc -I../.. -Igen -I../../third_party/flatbuffers/include -fno-strict-aliasing --param=ssp-buffer-size=4 -fstack-protector -funwind-tables -fPIC -B../../third_party/binutils/Linux_x64/Release/bin -pthread -fcolor-diagnostics -fmerge-all-constants -Xclang -mllvm -Xclang -instcombine-lower-dbg-declare=0 -no-canonical-prefixes -flto=thin -fwhole-program-vtables -fcomplete-member-pointers -m64 -march=x86-64 -O2 -fno-ident -fdata-sections -ffunction-sections -fno-omit-frame-pointer -g1 -fsanitize=cfi-vcall -fsanitize-blacklist=../../tools/cfi/blacklist.txt -fsanitize=cfi-icall -fvisibility=hidden -Wheader-hygiene -Wstring-conversion -Wtautological-overlap-compare -Wall -Wno-unused-variable -Wno-missing-field-initializers -Wno-unused-parameter -Wno-c++11-narrowing -Wno-covered-switch-default -Wno-unneeded-internal-declaration -Wno-undefined-var-template -Wno-address-of-packed-member -Wno-nonportable-include-path -Wno-user-defined-warnings -Wno-unused-lambda-capture -Wno-null-pointer-arithmetic -Wno-enum-compare-switch -Wno-ignored-pragma-optimize -Wno-exit-time-destructors -Wno-header-hygiene -Wno-return-type -std=gnu++14 -fno-exceptions -fno-rtti -nostdinc++ -isystem../../buildtools/third_party/libc++/trunk/include -isystem../../buildtools/third_party/libc++abi/trunk/include --sysroot=../../build/linux/debian_sid_amd64-sysroot -fvisibility-inlines-hidden -c ../../third_party/flatbuffers/src/flatc.cpp -o obj/build_extra/flatbuffers/compiler_files/flatc.o
clang++: error: no such file or directory: '../../tools/cfi/blacklist.txt'
[10/871] CXX obj/build_extra/flatbuffers/compiler_files/idl_gen_grpc.o
ninja: build stopped: subcommand failed.`

**Comment by ry** [comment]
I've done some more research into this warning. It seems to be caused by old versions of ccache (before 3.3) which didn't recognize `-isystemPATH` but only `-isystem PATH`. [Unfortunately `//build` concatenates the arguments](https://cs.chromium.org/chromium/src/build/config/posix/BUILD.gn?l=53&rcl=a0c5484dfce884bf6ce702147399556faa84500e).

Initially I updated a commit to disable `use_custom_libcxx` and `use_sysroot` as a workaround. But that ends up introducing even more warnings (lots of unreachable code). However once I updated my ccache I get hardly no warnings. Therefore I will back out of the changes to `use_custom_libcxx` and `use_sysroot`.

Unfortunately travis has the old version of ccache. So, I will just leave things as they are for now.
